tests/publiccloud/check_registercloudguest.pm

```diff
@@ -39,80 +39,75 @@ sub run {
         # all BYOS related checks. So we just regestering system and going further
         registercloudguest($instance);
     } elsif (is_byos()) {
-        if ($instance->run_ssh_command(cmd => 'sudo zypper lr', proceed_on_failure => 1) !~ /No repositories defined/gm) {
+        if ($instance->ssh_script_output(cmd => 'sudo zypper lr') !~ /No repositories defined/gm) {
```

Comment:
you did a VR for PAYG image , please clone BYOS one . I suspect that you will get unexpected behavior here ...

Reply:
https://openqa.suse.de/tests/10106797#